.github/workflows/integration-tests-2024.1.21-IPV4.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
                 ssl-enabled: ${{ env.ssl-enabled }}
                 tablets: ${{ env.tablets }}
             - name: Run tests
-              run: make pkg-integration-test IP_FAMILY=${{ env.ip-family }} SSL_ENABLED=${{ env.ssl-enabled}} BACKUP_METHOD=${{ env.backup-method }} RESTORE_METHOD=${{ env.restore-method }} PKG=./pkg/service/backup
+              run: make pkg-integration-test TABLETS=${{ env.tablets }} IP_FAMILY=${{ env.ip-family }} SSL_ENABLED=${{ env.ssl-enabled}} BACKUP_METHOD=${{ env.backup-method }} RESTORE_METHOD=${{ env.restore-method }} PKG=./pkg/service/backup
```

Comment:
In general, we have a simple script for generating workflows with tests against different scylla versions. In order to modify them, `.github/cfg/integration-test-core.yaml` should be adjusted and `go run .github/cfg/main.go` should be executed.